### DISCUSSION TITLE
config/plugins/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#331] [docs-comment]
# config/plugins/

主题内置的使用的插件，扩展了主题的众多功能，你可以在 plugins 配置中， 实现对内部使用的各个插件的自定义配置。 配置 所有主题内部使用的插件， 均在 plugins 字段中进行配置。 提示 您无需重复安装这些内置插件，也无需在 vuepress配置 > plugins 中添加它们。主题已在内部完成了这些工作。

https://theme-plume.vuejs.press/config/plugins/

**Comment by sunshang-hl** [comment]
可否给代码块标上文件名称?便于看教程的去对应的文件学习和修改?

**Reply by pengzhanbo** [reply]
已更新。

**Comment by c2ccc2** [comment]
有没有放统计的插件

**Reply by pengzhanbo** [reply]
<https://ecosystem.vuejs.press/zh/plugins/analytics/>

**Reply by c2ccc2** [reply]
收到，感谢。
